### PR TITLE
Search and filter

### PR DESCRIPTION
**Pull request by StevenPintoVC**
More integrated HAZZAH

### CODE REVIEW

**Review by AnneMienBBD — APPROVED** [review]

File: app/public/css/pop-up.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+
+  background: rgba(0, 0, 0, 0.29);
+  ;
```
Comment (JaredSwanzen) [inline app/public/css/pop-up.css]: What on earth is this!?

File: app/public/js/search.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const name = document.getElementById( "nameInput" );
+const surname = document.getElementById( "surnameInput" );
+const team = document.getElementById( "teamInput" );
+const Skills = document.getElementById( "skillsInput" );
+
+//This will then call the backend api to add a dev to the db
+function addDevSubmit () {
+  console.log(document.getElementById("nameInput"));
+
+  // const selectedSkills = [ ...Skills.options ].filter( option => option.selected ).map( option => option.value );
```
Comment (JaredSwanzen) [inline app/public/js/search.js]: Naughty naughty

File: app/views/partials/availabilityPopUp.ejs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <label for="endInput">End Date:</label>
+    <input type="date" required id="endInput" />
+    <label for="hourInput">Hours:</label>
+    <input type="number" min=1 required id="hourInput" />
+    <button id="cancelButton">Cancel</button>
+    <button id="acceptButton">Accept</button>
+  </section>
+</article>
+
+<script>
```
Comment (JaredSwanzen) [inline app/views/partials/availabilityPopUp.ejs]: Naughty naughty

File: app/views/partials/successPopUp.ejs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      Successful!
+    </h1>
+    <h5 class="pop-up-label">
+      <%=myLabel%>
+    </h5>
+    <button id="success-button" onclick="buttonHandler()">Okay</button>
+  </section>
+</article>
+
+<script>
```
Comment (JaredSwanzen) [inline app/views/partials/successPopUp.ejs]: Naughty naughty

**Review by JaredSwanzen — CHANGES_REQUESTED** [review]
Te naughties

**Review by JaredSwanzen — APPROVED** [review]

**Pull request merged**